src/components/FormValidator.js

```diff
@@ -88,7 +88,7 @@ export default class FormValidator {
   enableValidation() {
     this._formElement.addEventListener("submit", (evt) => {
       evt.preventDefault();
-      this.disableSubmitButton();
+      if (this._inputList.length !== 0) this.disableSubmitButton();
```

Comment:
```js
      if (this._inputList.length !== 0) this.disableSubmitButton();
```


Please, delete it from the submit handler

You need to **disable** the button only in a `then` block after a **successful response** to be able to click it again if there is a server error.